tutorials/gm-world.md

```diff
@@ -68,10 +68,11 @@ If you get errors of `gmd` not found, you may need to add the `go/bin` directory
 :::
 
 ```bash
-curl -sSL https://rollkit.dev/install-gm-rollup.sh | sh
+cd $HOME
```

Comment:
@yarikbratashchuk I think this needs sudo access. [see without sudo logs](https://app.warp.dev/block/oD5RrlXSSpcteFcEpVXuXU)

Reply:
changed init.sh script to go install, so we don't need sudo now, as it's installed into the path
sudo is another dependancy we would require to install

Reply:
sounds good 🫡